templates.json

```diff
@@ -702,9 +690,34 @@
         "path": "editor-extension-collection"
       }
     ],
-    "organizationBetaFlags": [
-      "editor_extension_collection"
-    ]
+    "organizationBetaFlags": ["editor_extension_collection"]
+  },
+  {
+    "identifier": "discount_details_ui",
+    "name": "Discount Details - UI Extension",
+    "defaultName": "discount-details-ui",
+    "group": "Admin",
+    // TODO is this correct?
+    "supportLinks": ["https://shopify.dev/docs/apps/discounts"],
+    // TODO is this correct? It is a private repo currently
+    "url": "https://github.com/Shopify/discount-extensions-examples",
```

Comment:
I also feel we shouldn't have needed two repos, one for development and one for the tutorial.. I think it would be great if we could consolidate our efforts into one repo again 🙏 @devisscher @nova-srahman

Reply:
Agree, one repo would be ideal :)